services/minespace-web/src/styles/settings/variables.scss

```diff
@@ -55,8 +56,8 @@ $danger-btn-color-disabled: rgba($danger-btn-color, 0.5);
  */
 
 // Buttons
-$primary-btn-color: $violet;
-$primary-btn-color-hover: color.adjust($primary-btn-color, $lightness: -20%);
+$primary-btn-color: $gov-blue;
+$primary-btn-color-hover: $gov-blue-hover;
```

Comment:
no purple buttons on MS!
Also the darkening hover function turned gov-blue black so I just grabbed whatever was already there and made it a variable.